rego-templates/embedded.go

```diff
@@ -0,0 +1,101 @@
+package rego_templates
```

Comment:
'rego-templates' and 'rego-filters' folders contain rego files only. Rego related go code is in `regoservice` package

Reply:
there is no way to embed files with relative path